app-frontend/src/app/components/selectToolModal/selectToolModal.html

```diff
@@ -0,0 +1,39 @@
+<div class="modal-header">
+  <span class="badge"><i class="icon-bucket"></i></span>
```

Comment:
Since buckets are projects, should this be something like `wrench`? I was also going to suggest `cogs` but those look like settings